### ISSUE TITLE
Better support for complete novices

### ISSUE THREAD

**Issue by gabekneisley** [education/teachers#4]
I love the concept, but where does a novice go? The whole project presupposes a level of knowledge and enthusiasm that some readers may not have yet. For example, a reader needs to create an issue (basically complain) to join a discussion. I'd suggest a welcome video, a friendly link to what github is and what you do, and a thorough explanation of why github is so important as an educational and critical thinking tool.

Specifically, I mean something like:

_GitHub is a software company dedicated to helping people organize, annotate, and share knowledge. Our technology is built upon, and heartily embraces open source software. In fact, XX% of the world's open source projects use GitHub to keep things moving smoothly. We believe our expertise at organization, annotation, and sharing can work for education as well as it has for computer programming...._

Basically, if it were explained more thoroughly, I think it would encourage sharing because it would be a painless read for those new to some of the concepts, and a solid grounding in how to explain it externally for those in the know.

**Comment by dpentecost** [comment]
Just wanted to agree with the commenter, even after more than a year with no further advances in this discussion.

gabekneisley is dead on - the semantics of GitHub are difficult to get past, the obstacles for novices are still there, most tutorials assume an understanding before you begin that just does not exist outside of computer science classes. It can be surmounted by obsessive googling and sheer persistence, but the evident usefulness of a shared repository in a wide range of fields calls for an effort to make it more accessible. I am tackling that challenge and I hope that gabekneisley also keeps working at it.
